dev/examples/controls/line.js

```diff
@@ -427,3 +427,20 @@ export const renderNoDataView = (id) => {
     );
     return lineDefault;
 };
+export const renderLineCustomPadding = (id) => {
+    const data = utils.deepClone(getDemoData(`#${id}`, "LINE_DEFAULT"));
+    data.showLegend = false;
+    data.axis.x.show = false;
+    data.axis.y.show = false;
+    data.padding = {
```

Comment:
nit: The order of the direction can be in the same way we define padding property in CSS i.e. in order of TRBL (top , right, bottom and then left)

Reply:
Its an object, the order is not guaranteed anyway.